storagegroup/test/generate.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // StorageGroup returns random storagegroup.StorageGroup.
 func StorageGroup() storagegroup.StorageGroup {
-	x := storagegroup.New(322, checksumtest.Checksum(), oidtest.IDs(2))
+	n := 1 + rand.Int()%10
```

Comment:
`Intn()` would be better even though it's test code and we don't care much about value distribution.

Reply:
done